.github/workflows/ui-release-image-stable.yaml

```
@@ -68,25 +71,79 @@
+          tags: |
+            ${{ github.event.inputs.version }}
       - name: Login to GitHub Container Registry
         uses: docker/login-action@v3
         with:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
-      - name: Build and push image
+      - name: Create manifest list and push
+        working-directory: /tmp/digests
         run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2046:warning:2:33: Quote this to prevent word splitting

(shellcheck)